molecule/pki/verify.yml

```diff
@@ -46,12 +46,18 @@
 
     - name: Check in logs a client did connect
       ansible.builtin.slurp:
-        src: /var/log/alfresco/ats-ate-aio.log
-      register: aio_log
+        src: /var/log/alfresco/{{ item.file }}.log
+      register: ats_log
```

Comment:
static register + loop doesn't seems a good idea

Reply:
why not? The tests I made locally seems to prove good results as long as you realise it register the current iteration rather than the list of iterations (which you get once out of the loop)

Reply:
oh ok it's used only during the loop and not after - so it make sense